README.rst

```diff
@@ -37,6 +37,82 @@ on `PyPI`_ and `GitHub`_.
 .. _`PyPI`: https://pypi.org/project/pymedphys/
 .. _`GitHub`: https://github.com/pymedphys/pymedphys
 
+Statement of need
```

Comment:
Can we have this elsewhere within the documentation, and simply link to it within the readme?